src/test/java/com/github/brandtg/PlotTest.java

```diff
@@ -35,6 +38,17 @@ public void testPlot() throws Exception {
 
     final StlDecomposition stl = new StlDecomposition(config);
     final StlResult res = stl.decompose(ts, ys);
+
+    final File output = new File("seasonal.png");
+    final File hourly = new File("stl-hourly.png");
+
+    StlPlotter.plot(res, "New Title", Hour.class, hourly);
+    StlPlotter.plot(res, output);
     StlPlotter.plot(res);
```

Comment:
I think we need to remove the `StlPlotter.plot` calls that do not output to a file, as the travis build fails with the following error still

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running TestSuite
Configuring TestNG with: TestNG652Configurator
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 6.897 sec <<< FAILURE! - in TestSuite
testPlot(com.github.brandtg.PlotTest)  Time elapsed: 3.015 sec  <<< FAILURE!
java.awt.HeadlessException: 
No X11 DISPLAY variable was set, but this program performed an operation which requires it.
	at java.awt.GraphicsEnvironment.checkHeadless(GraphicsEnvironment.java:207)
	at java.awt.Window.<init>(Window.java:536)
	at java.awt.Frame.<init>(Frame.java:420)
	at javax.swing.JFrame.<init>(JFrame.java:225)
	at org.jfree.ui.ApplicationFrame.<init>(ApplicationFrame.java:65)
	at com.github.brandtg.StlPlotter$ResultsPlot.<init>(StlPlotter.java:61)
	at com.github.brandtg.StlPlotter.plot(StlPlotter.java:42)
	at com.github.brandtg.PlotTest.testPlot(PlotTest.java:45)
```